net-wireless/gr-sdrplay3/gr-sdrplay3-20230421.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit cmake python-single-r1
+
+DESCRIPTION="GNU Radio OOT module for SDRplay RSP devices - SRDPlay API v3"
+HOMEPAGE="https://github.com/fventuri/gr-sdrplay3"
+
+if [[ ${PV} == 9999* ]]; then
```

Comment:
please fix the whitespace issues in this block

Reply:
this if block has uneven indent

Reply:
what are you smoking? I want some too.
all indents are 1 tab long.

Reply:
nevermind, I'm blind

Reply:
![Screenshot_2023-06-08_16-25-35](https://github.com/pentoo/pentoo-overlay/assets/1670905/9660a8bf-73dd-4148-a774-dd7fb2b75096)

Reply:
that's even on your screen?

Reply:
fixed, thanks.